homeassistant/components/plugwise/entity.py

```diff
@@ -72,6 +72,16 @@ def device(self) -> dict[str, Any]:
         """Return data for this device."""
         return self.coordinator.data.devices[self._dev_id]
 
+    @property
+    def devices(self) -> dict[str, dict[str, Any]]:
+        """Return devices data."""
+        return self.coordinator.data.devices
+
+    @property
+    def gateway(self) -> dict[str, Any]:
+        """Return gateway data."""
+        return self.coordinator.data.gateway
```

Comment:
I don't think these are needed or necessary, they only make code harder to read (and it adds more layers).

In case of the above `device` property, I think it makes sense (as it brings in a local protected variable to find the right one), but for these two they just add overhead.